src/Plugin/EntityReferenceSelection/OgSelection.php

```diff
@@ -25,7 +25,7 @@
  * @EntityReferenceSelection(
  *   id = "og:default",
  *   label = @Translation("OG selection"),
- *   group = "default",
+ *   group = "og",
```

Comment:
> OK, so if we are doing that, then we DEFINITELY need the below change to not throw anthing

Can you provide ways to reproduce the problem - I'd like to better understand the scenario, I have a feeling I'm missing your point ;)

> Where would that behaviour of always selecting the of handler get used?

https://github.com/amitaibu/og/blob/8.x-1.x/src/Plugin/EntityReferenceSelection/OgSelection.php#L63

We get the "original" (entity specific) query, and then we add conditions to it.